charts/dispatch/charts/identity-manager/values.yaml

```diff
@@ -17,7 +17,7 @@ service:
   externalPort: 80
   internalPort: 80
 oauth2proxy:
-  image: docker.io/neosab/oauth2-proxy
```

Comment:
We need to put this in better place...  @kars7e just created a dispatchframework org in docker.

Reply:
Yup, agree, I will open an issue to track and migrate it from my docker account.